### DISCUSSION TITLE
Demon: The Fallen Earthbound

### DISCUSSION THREAD

**Discussion by PeterMcD-stack** [JohanFalt/Foundry_WoD20#956] [Q&A] [answered]
Hello, any chance to change convictions to urges on the demon sheet for earthbound demons? changing Conviction, self control and courage to Flesh, Thought and Emotion.

**Comment by JohanFalt** [comment]
Will add for coming version.

**Comment by JohanFalt** [comment]
I assume this is for NPC?

**Comment by PeterMcD-stack** [comment]
for an npc yes, but could run an earthbound game at some point I suppose

**Comment by JohanFalt** [comment, marked as answer]
Will be present in version 4.2
